sdk/avalon_sdk/ethereum/ethereum_worker_registry.py

```diff
@@ -189,21 +188,18 @@ def worker_register(self, worker_id, worker_type, organization_id,
         """
 
         if (self.__contract_instance is not None):
-            if not is_valid_hex_str(binascii
-                                    .hexlify(worker_id).decode("utf8")):
+            if not is_valid_hex_str(worker_id):
                 logging.error("Invalid worker id {}".format(worker_id))
                 return None
             if not isinstance(worker_type, WorkerType):
                 logging.error("Invalid workerType {}".format(worker_type))
                 return None
-            if not is_valid_hex_str(binascii
-                                    .hexlify(organization_id).decode("utf8")):
+            if not is_valid_hex_str(organization_id):
                 logging.error("Invalid organization id {}"
                               .format(orgnization_id))
```

Comment:
The variable is misspelled and the error message will fail.
Change orgnization_id to organization_id.